k8s/kustomize/helm-components-sc/patch_envoy_json6902.yaml

```diff
@@ -1,3 +1,3 @@
 - op: replace
-  path: /spec/template/spec/securityContext
+  path: /spec/components/4/podSpec/securityContext
```

Comment:
can you add a note somewhere on how to get the index so that it can be changed in the future if needed?